<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>550</width>
      <height>375</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseSrcFile">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Source File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseBgFile">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseExFile">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetup">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrain">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameCountLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>frameCount</string>
    </property>
   </widget>
   <widget class="QLabel" name="constFrameLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FrameCount:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
